server/routes/explore/helpers.py

```diff
@@ -96,7 +96,8 @@ def parse_query_and_detect(request: Dict, backend: str, client: str,
   i18n = i18n_str and i18n_str.lower() == 'true'
 
   # Index-type default is in nl_server.
-  embeddings_index_type = request.args.get(params.Params.INDEX.value, '')
+  idx_param_str = request.args.get(params.Params.INDEX.value, '')
```

Comment:
nit: should we do request.args.getList & have the clients pass the values as a list